packages/admin-panel/src/editor/utils.js

```diff
@@ -0,0 +1,22 @@
+/*
+ * Tupaia
+ * Copyright (c) 2017 - 2024 Beyond Essential Systems Pty Ltd
+ */
+
+/**
+ * This is a utility function that returns the key to use when editing a field. This is useful because sometimes if the data is pre-filled from the api, the key of the data is different from the key that should be used when editing the field.
+ */
+export const getFieldEditKey = field => {
```

Comment:
I don't love the use of this around the admin panel, and I tried to make it make sense, where I used it, but I couldn't think of a way to remove this altogether, or even keep it to being set in one place, so I decided a big refactor of it wasn't worth it and we can just use this reusable function